src/fastoad/notebooks/03_custom_modules/beam_problem.ipynb

```diff
@@ -0,0 +1,271 @@
+    "The objectives this tutorial will be to create and call custom modules through FAST-OAD to solve the following problems:\n",
+    "* For **F** = 1000N, **L** = 2.0m, **l** = 0.3m find the height **h** that leads to a displacement of 0.02m considering an aluminium beam. \n",
+    "* For **F** = 1e5N, **L** = 10.0m, **l** = 0.3m find the height **h** so that the maximum normal stress within the beam do not exceed the aluminium yield stress (450e6 Pa)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"row\">\n",
+    "  <div class=\"column\">\n",
+    "    <img src=\"./img/problem_description.png\" width=\"600\">\n",
+    "  </div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os.path as pth\n",
+    "import logging\n",
+    "import fastoad.api as oad"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Directories and files definition "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_FOLDER = \"data\"\n",
+    "\n",
+    "WORK_FOLDER = \"workdir\"\n",
+    "\n",
+    "CONFIGURATION_FILE_NAME = pth.join(DATA_FOLDER, \"beam_problem.yml\")\n",
+    "\n",
+    "logging.basicConfig(level=logging.INFO, format='%(levelname)-8s: %(message)s')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Custom Modules\n",
+    "\n",
+    "The following modules are created: [section_properties.py](./modules/section_properties.py), [weight.py](./modules/weight.py), [displacements.py](./modules/displacements.py), [stresses.py](./modules/stresses.py). They respectively compute the heigth of the beam for a given moment of inertia and width, the linear weight **w** from material density and section dimensions, the necessary moment of inertia to reach a given displacements and the necessary moment of inertia not to overcome yield stress. \n",
+    "\n",
+    "As you may notice, each module is registered using the decorator `@oad.RegisterOpenMDAOSystem`. \n",
+    "\n",
+    "An example is provided for a geometric module that compute the height of the beam for a given moment of inertia and a given width: \n",
+    "\n",
+    "<div class=\"row\">\n",
+    "    <div class=\"column\">\n",
+    "        <img src=\"./img/custom_module_ex.gif\" width=\"600\">\n",
+    "  </div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Configuration file\n",
+    "Once the modules are created, the problem can be defined through the [configuration file](./data/beam_problem.yml): \n",
+    "* First, the folder containing your custom module must be indicated within the `module_folders` section.\n",
```

Comment:
1. The sequence sounds strange to me: "first, ... **must**..., then your problem **is**....". I would simply remove "first" and "then"
2. I would put "The folder(s) containing..." for sake of genericity
3. "...containing your custom modules..." because there are four of them.